src/components/DisplayComponent.js

```diff
@@ -1,6 +1,16 @@
 import React, { Component } from 'react';
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend
+} from 'recharts';
 import '../styles/DisplayComponent.css';
 import Filters from './Filters';
+import testTable from '../../src/testStuff/cities.json';
```

Comment:
This json data is not used anywhere, yet

Reply:
Ok! Will remove!